Whiskey/Tasks/Invoke-WhiskeyPipelineTask.ps1

```diff
@@ -0,0 +1,105 @@
+ function Invoke-PipelineTask 

```

Comment:
Per the other tasks in Whiskey, should this be named `Invoke-WhiskeyPipelineTask`?